contracts/nft/PFPAuction.sol

```diff
@@ -207,6 +207,8 @@ contract PFPAuction is OwnableUpgradeable, ReentrancyGuardUpgradeable, IERC721Re
   }
 
   function bid(address tokenAddress, uint256 tokenId, uint256 expectedSpending) external payable nonReentrant {
+    uint256 price = getNextPrice(tokenAddress, tokenId);
+    _lfg.approve(address(this), price);
```

Comment:
What is this?
The contract cannot approve itself for spending users' LFG; with this command, it is approving itself to spend its own LFG, which I don't think is what you want.

Reply:
Got it, I misunderstood the Approve method